### PR TITLE
I think you mean _DAN_ book.

### PR DESCRIPTION
**Pull request by dsberger**

### CODE REVIEW

**Comment by Schwad** [comment]
SchwadBook

File: homepage.html [hunk 1]
```diff
@@ -0,0 +1,186 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
```
Comment (Schwad) [inline homepage.html:5]: The meta tags here are good, did you pull them from the bootstrap docs pages? I will have to re-learn what these do and how they function.

File: homepage.html [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+
+  <body>
+
+    <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
+    <!-- Include all compiled plugins (below), or include individual files as needed -->
+    <!-- Latest compiled and minified JavaScript -->
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.1/js/bootstrap.min.js"></script>
+
+    <nav class="navbar navbar-default navbar-fixed-top">
```
Comment (Schwad) [inline homepage.html:30]: Good use of the bootstrap navbar class. I think I just blunt-forced my nav so that's good. I wonder if the settings fix the page to the top if it's a long page and you scroll? If you don't want it to stick to the top of the page when you scroll that might be a consideration. I will have to check the documentation to see the effects of this on a long page.

File: homepage.html [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+              <input type="email" class="form-control" placeholder="Email">
+              <input type="password" class="form-control" placeholder="Password">
+            </div>
+            <button type="submit" class="btn btn-default">LOG IN</button>
+          </form>
+        </div><!-- /.navbar-collapse -->
+      </div><!-- /.container-fluid -->
+    </nav>
+
+    <main class="container">
```
Comment (Schwad) [inline homepage.html:56]: Good semantic use of main.

File: homepage.html [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+      <div class="row">
+        <div class="home-page-left col-sm-6">
+          <h3>Connect with all your friends!</h3>
+          <ul class="things-to-do-with-friends">
+            <li>See photos and update your news feed.</li>
```
Comment (Schwad) [inline homepage.html:61]: Good class title. Oh man this is an efficient way to put together this session.

File: homepage.html [hunk 5]
```diff
@@ -0,0 +1,186 @@
...
+                  <option value="26">26</option>
+                  <option value="27">27</option>
+                  <option value="28">28</option>
+                  <option value="29">29</option>
+                  <option value="30">30</option>
+                  <option value="31">31</option>
+                </select>
+                <select name="year" id="year">
+                  <option disable selected>Year</option>
+                  <option value="2000">2000</option>
```
Comment (Schwad) [inline homepage.html:156]: It's just a barrel of fun putting in all those values for the options. I wonder if bootstrap has a less brute force way of doing this.

File: styles.css.scss [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+  }
+
+  button {
+    width: 100%;
+    height: 3em;
+    font-size: 1.5em;
+  }
+}
+
+form>div {
```
Comment (Schwad) [inline styles.css.scss:57]: Nice use of ">"

File: styles.css.scss [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+  form>div {
+    margin: 0;
+  }
+  button {
+    font-size: 0.8em;
+  }
+}
+
+.home-page-left {
+  padding-top: 150px;
```
Comment (Schwad) [inline styles.css.scss:22]: I don't think we are ever going to get out of having to use padding and margins on some level to make things work right.

File: homepage.html (hunk 1 above)
Comment (dsberger) [inline homepage.html:5]: Yes, I started with the baseline template from Bootstrap and these are part of that. Have to add the CSS and JS links they give also.

File: homepage.html (hunk 5 above)
Comment (dsberger) [inline homepage.html:156]: Cut-n-paste from here! https://snipt.net/tseven/month-day-html-dropdown-list/
Bootstrap might have something fancy though.
